blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -518,6 +518,8 @@ function handle_customize_save_after( $wp_customize ) {
 		$user_theme_json_post_content->version                     = 1;
 		$user_theme_json_post_content->isGlobalStylesUserThemeJSON = true;
 
+		$this->unset_property_if_it_exists( $user_theme_json_post_content->settings->typography, 'fontFamilies' );
```

Comment:
We should add a comment to explain what this is for

Reply:
comment added.